code/datums/components/nanites.dm

```diff
@@ -56,6 +57,7 @@
 		RegisterSignal(parent, COMSIG_MOVABLE_HEAR, .proc/on_hear)
 		RegisterSignal(parent, COMSIG_SPECIES_GAIN, .proc/check_viable_biotype)
 		RegisterSignal(parent, COMSIG_NANITE_SIGNAL, .proc/receive_signal)
+		RegisterSignal(parent, COMSIG_PARENT_EXAMPLE, .proc/on_examine)
```

Comment:
Oh, one last thing. carbon, human and silicon examine functions won't check the parent versions and thus won't send the signal, making this totally useless.
Find a solution out of this problem, though even just adding SEND_SIGNAL() manually may do it.